src/blr/Snakefile

```diff
@@ -168,12 +168,13 @@ rule bam_to_fastq:
         r2_fastq = "reads.2.final.fastq.gz"
     input:
         bam = "mapped.sorted.tag.mkdup.bcmerge.mol.filt.bam"
-    log: "picard_samtofastq.log"
+    log: "samtools-fastq.log"
     shell:
-        "picard -Xms{config[heap_space]}g SamToFastq"
-        " I={input.bam}"
-        " FASTQ={output.r1_fastq}"
-        " SECOND_END_FASTQ={output.r2_fastq} 2>> {log}"
+        "samtools fastq"
+        " -@ {threads}"
```

Comment:
For `threads` to be useful we need to define the maximum number of threads for this rule. Add `threads: 20` after `log: "samtools-fastq.log"`

Reply:
I just remembered, it's fixed now.